src/reference/crash-reporting.md

```diff
@@ -25,18 +25,27 @@ over time. A crash report may contain the following information:
 Google handles all data reported by this tool in accordance with the 
 [Google Privacy Policy][].
 
+Additionally, you may review the recently reported data in the 
+`.dart-tool/dart-flutter-telemetry.log` file under the home directory
+`~/` (macOS/Linux) or the `<username>\AppData` (Windows) directory. 
```

Comment:
**issue:** This points to the wrong Windows directory. The macOS directory is correct. Make this change and I can approve.
```suggestion
You may review the recently reported data in the 
`.dart-tool/dart-flutter-telemetry.log` file.
In macOS or Linux, this log is located in the home directory (`~/`).
In Windows, this log is located in the Roaming AppData directory in Windows (`%APPDATA%`). 
```

Reply:
Nice catch to you use the env variable, that's actually how we do it in `package:unified_analytics` as well

```dart
Directory getHomeDirectory(FileSystem fs) {
  String? home;
  Map<String, String> envVars = io.Platform.environment;

  if (io.Platform.isMacOS) {
    home = envVars['HOME'];
  } else if (io.Platform.isLinux) {
    home = envVars['HOME'];
  } else if (io.Platform.isWindows) {
    home = envVars['AppData'];
  }

  return fs.directory(home!);
}
```